### DISCUSSION TITLE
Blackhole, OBS, Zoom

### DISCUSSION THREAD

**Discussion by ManuelHeche1983** [ExistentialAudio/BlackHole#281] [Q&A]
Hi! I‘m new to blackhole and hopefully get helped with my special issue... I‘m a guitar teacher and using OBS Virtual Cam into Zoom, so I can use multiple cams and views.

I‘m using a Mic into Input 1 and a guitar (pedalboard, mixer, headphone out) into Input 3/4 of my NI Komplete Audio 6 connected to my MBP (Mid 2014, BIG SUR).

Using LadioCast Mixer works pretty good for that. But now I want to change 2 things. 

1st I want to integrate Desktop Audio, so my students and I can hear it, without creating an audio loop. 

2nd I want to put a latency on the audio signal, that goes to the students, to synchronize video and audio signal.

I appreciate your help!

Manu

**Comment by HC21Stream** [comment]
Hi Manu, i got the same problem--- temporary I use built in output by aux output 1  and set the volume little bit down, so  the loop is not that loud.

Hans
